selfservice/flow/registration/error.go

```diff
@@ -23,6 +23,7 @@ var (
 	ErrHookAbortFlow        = errors.New("aborted registration hook execution")
 	ErrAlreadyLoggedIn      = herodot.ErrBadRequest.WithID(text.ErrIDAlreadyLoggedIn).WithError("you are already logged in").WithReason("A valid session was detected and thus registration is not possible.")
 	ErrRegistrationDisabled = herodot.ErrBadRequest.WithID(text.ErrIDSelfServiceFlowDisabled).WithError("registration flow disabled").WithReason("Registration is not allowed because it was disabled.")
+	ErrDuplicateCredentials = herodot.ErrBadRequest.WithError(text.NewErrorValidationDuplicateCredentials().Text)
```

Comment:
This could maybe be specific to OIDC only? 
So this type must be `ErrDuplicateOIDCCredentials`?

A new error could also be created for this specific case of duplicate account on oidc registration to provide better information to the user.